frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.tsx

```diff
@@ -420,7 +459,7 @@ export function ObjectDetailHeader({
       </div>
 
       {showControls && (
-        <Flex align="center" gap="xs" p="md">
+        <Flex align="center" gap="0.5rem" p="1rem">
```

Comment:
Reason for this line change: https://github.com/metabase/metabase/pull/32331#discussion_r1267827835